gcloud-java-compute/src/test/java/com/google/gcloud/compute/ImageDiskConfigurationTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import com.google.gcloud.compute.DiskConfiguration.Type;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class ImageDiskConfigurationTest {
+
+  private static final Long SIZE = 42L;
+  private static final DiskTypeId DISK_TYPE = DiskTypeId.of("project", "zone", "type");
+  private static final ImageId IMAGE = ImageId.of("project", "image");
+  private static final String IMAGE_ID = "imageId";
+  private static final ImageDiskConfiguration DISK_CONFIGURATION =
+      ImageDiskConfiguration.builder(IMAGE)
+          .sizeGb(SIZE)
+          .diskType(DISK_TYPE)
+          .sourceImageId(IMAGE_ID)
+          .build();
+
+  @Test
+  public void testToBuilder() {
+    compareImageDiskConfiguration(DISK_CONFIGURATION, DISK_CONFIGURATION.toBuilder().build());
+    ImageId newImageId = ImageId.of("newProjet", "newImage");
```

Comment:
Nit: doesn't really matter for test correctness, but "newProjet" --> "newProject"